pyRMSD/symmTools.py

```diff
@@ -54,6 +71,46 @@ def symm_permutations(groups):
             yield [swapped_head] + tail_permutation
     else:
         yield []
+
+def symm_permutations_new(groups):
+    """
+    Generator that produces the possible permutations of the symmetry groups.
+    Suposing we have this symmetries definition:
+        [
+            [ [1,2,0] ], 
+            [ [3,4],[5,6] ]
+        ]
+    Meaning that coordinates 1 and 2 and 3 are equivalent, and that 3 and 4, as well as 5 and 6 are equivalen and must be changed 
```

Comment:
Don't you mean "coordinates 1 and 2 and **0** are equivalent" ?

Reply:
Fixed in the new PR